Improve webui password reset text (#9)
UmbrelInfo/UmbrelInfo.github.io: troubleshooting.md

Do this only if you **do not have any funds** on your LND wallet!

> You are going to loose seed / settings / user data / application installed!

Reviewer: I think this is better, @louneskmt what do you think?
```suggestion
You are going to loose your seed, settings, data and applications!
```

Reviewer 2: Also change the question to something like "Reset your Umbrel user data (if you lost your password)". We can also add a little sentence to tell the user that already has funds to keep its seed and backup file to be able to restore them later on.

Author: Updated, feel free to check it :)
